js/stories.js

```diff
@@ -61,7 +61,7 @@ function getStar(story) {
    <i class="${favorite} fa-star"></i>
    </span>`
 }
-
+/**Show users favorite stories on page */
```

Comment:
Useful comment, I just have feedback on the [commit message itself](https://github.com/SpringBoardRepo/Hack-0r-snooze/pull/1/commits/ba8d1bb7bb522d1d070ebf02672b893167d1ac1d). You wrote `empty fav.list before adding new` - this doesn't make sense to me. For this commit I would put something like this as a message: `added description to function`.